tools/webassembly.py

```diff
@@ -162,6 +162,12 @@ class DylinkType(IntEnum):
   IMPORT_INFO = 4
 
 
+class TargetFeaturePrefix(IntEnum):
+  USED = 0x2b
+  DISALLOWED = 0x2d
+  REQUIRED = 0x3d
```

Comment:
Do we still emit `REQUIRED`? I can't find anywhere in LLVM we emit it after https://github.com/llvm/llvm-project/commit/3f34e1b883351c7d98426b084386a7aa762aa366. cc @tlively

Reply:
Not sure. Seems OK to include it in this enum though. For now we really only check for USED in the test.

Reply:
This whole system is slightly overengineered (my fault 🫣), and I'm not aware of any uses of `REQUIRED`. It would be a nice cleanup to remove it entirely here and in the tool conventions doc.

Reply:
Done!
#22792
https://github.com/llvm/llvm-project/pull/113729
https://github.com/WebAssembly/tool-conventions/pull/236
https://github.com/WebAssembly/binaryen/pull/7034

Reply:
Amazing, thank you!